src/ct.rs

```diff
@@ -74,8 +72,7 @@ impl From<CtDecryptResult> for bool {
 
 impl<Aes, NonceSize> CtAeadDecrypt for AesGcm<Aes, NonceSize>
 where
-    Aes: BlockCipher<BlockSize = U16> + BlockEncrypt + NewBlockCipher,
-    Aes::ParBlocks: ArrayLength<Block<Aes>>,
+    Aes: BlockCipher + BlockSizeUser<BlockSize = U16> + BlockEncrypt,
```

Comment:
To mimic the change at https://github.com/RustCrypto/AEADs/compare/aes-gcm-v0.9.4...aes-gcm-v0.10.1#diff-5d410b74827da3f0ac941691bae50d4c3e370e9ae41ecbee7b5193223f42c8eeR205